package.json

```diff
@@ -32,16 +32,17 @@
         "vscode-languageclient": "1.3.1"
```

Comment:
Is it missing "vscode-jsonrpc" ? I cannot compile it without adding "vscode-jsonrpc" as a dependency.